### DISCUSSION TITLE
Confusing Versioning

### DISCUSSION THREAD

**Discussion by WeirdScienceX** [upscayl/upscayl#786] [Q&A]
Why have a 2.11 pre-release, but then the associated assets are 2.10.9

![image](https://github.com/upscayl/upscayl/assets/1296569/b16e3160-2b95-44d5-b0e0-f35024b15757)

**Comment by aaronliu0130** [comment]
This is the prerelease. It's one step away from the actual 2.11 release.

That said, @NayamAmarshe perhaps we could make it something like 2.11-pre next time?

**Reply by NayamAmarshe** [reply]
Not possible, unfortunately. 

2.10.9 has to be used in order to indicate 2.11's pre-release.

**Reply by aaronliu0130** [reply]
Why though? In semver, 2.11-pre would be before 2.11.

**Reply by NayamAmarshe** [reply]
> Why though? In semver, 2.11-pre would be before 2.11.

Node doesn't support -pre in package.json

**Reply by aaronliu0130** [reply]
Can it do -alpha?

**Reply by NayamAmarshe** [reply]
> Can it do -alpha?

No, it only supports numbers.

**Reply by aaronliu0130** [reply]
Just making sure that you've tried it? https://docs.npmjs.com/cli/v7/configuring-npm/package-json#version claims that anything parseable by https://github.com/npm/node-semver is fine, which seems to be able to parse -alpha by default.

**Reply by NayamAmarshe** [reply]
Ok, I'll give it a try next time :) Thanks!
